apypie/example.py

```diff
@@ -1,7 +1,7 @@
 import re
 
 
-EXAMPLE_PARSER = re.compile(r'(\w+)\s+([^\n]*)\n?(.*)?\n(\d+)\n(.*)', re.DOTALL)
+EXAMPLE_PARSER = re.compile(r'(\w+)\s+([^\n]*)\n?(.*)\n(\d+)\n(.*)', re.DOTALL)
```

Comment:
having `?` after a `(.*)` doesn't make sense anyways, because `?` is already included in the `*`